src/test/scala/com/timeout/healthchecks/HeathChecksTest.scala

```diff
@@ -49,6 +49,13 @@ class HeathChecksTest extends AnyFunSpec with ScalaFutures with Matchers with Ta
   }
 
   describe("Ping check") {
+
+    it("should return ok for valid database ping") {
+      whenReady(HealthChecks.pingDataBase("sdb.gp-db.dev.timeout.com", 3306, "sdbuser", "thepword").run()) { r =>
```

Comment:
this test may not have access to sdb.gp-db.dev.timeout.com, test should be mocked?

Reply:
I may just commented out. We can just run locally if needed. The code is simple so I was just a way to confirm the chnages

Reply:
I've removed the test